app/routes/products_.$productId/route.tsx

```diff
@@ -66,9 +66,28 @@ export default function ProductDetailsPage() {
                 className={styles.left}
             />
             <div className={styles.right}>
-                <div>{product.name}</div>
+                <div>
```

Comment:
is this an extra obsolete div ?

Reply:
no, because container (className={styles.right}) has display flex with defined gap, so I'm groupping name and sku in single group with that div